iu_xray_ontology.json

```diff
@@ -0,0 +1,53 @@
+
+{
+    "name": "iu_xray_ontology",
+    "additional_prefixes": [
+      "iu_xray.onto.iu_xray_ontology"
```

Comment:
The name is complicated, we can simply do `iu_xray.onto`?

Reply:
modified